doc/website/getting-started/overview.md

```diff
@@ -11,637 +10,290 @@ runtime and create _publishers_ and _subscribers_. The publishers send data of a
-handle a potential error in the ``or_else`` branch. If we wake up periodically, it is also possible that
-no data is received and if we want to handle this we can optionally do so in the ``if_empty`` branch.
+### Shared memory
 
-The usual case is that we actually receive data, and we process it in the ``and_then``. Notice that in the lambda we
-do not pass a ``CounterTopic`` directly but a reference to a ``iox::popo::Sample<const CounterTopic>&``. We can
-access the underlying ``CounterTopic`` either by getting a pointer to it via ``get`` or by using ``operator->``. In
-any case, we now can process or copy the received data and once the ``sample`` goes out of scope, the underlying
-``CounterTopic`` object is deleted as well (this happens when the temporary object returned by ``take`` is
-destroyed). This means it is only safe to hold references to the data as long as the ``sample`` exists. Should we
-need a longer lifetime, we have to copy or move the data from the ``sample``.
+To enable zero-copy inter-process communication, iceoryx uses the shared memory approach, i.e. publishers and 
+subscribers can communicate via shared memory resulting in zero-copy communication.
 
-This only allows us to get one sample at a time. Should we want to get all currently available samples we can do so
-by using an additional loop.
+Shared memory is physical memory that is made accessible to multiple processes via a mapping to a memory area in their 
+virtual address spaces.
 
+For further information have a look at our 
+[conceptual guide](https://github.com/eclipse-iceoryx/iceoryx/blob/master/doc/conceptual-guide.md).
+
+### Runtime
+
+Each application that wants to use iceoryx has to instantiate its runtime, which essentially enables communication
+with RouDi.
+
+To do so, the following lines of code are required
 ```cpp
-while (keepRunning)
-{
-    while (subscriber.hasData())
-    {
-        subscriber->take()
-            .and_then([](iox::popo::Sample<const CounterTopic>& sample) {
-                CounterTopic* ptr = sample.get();
-                /* process the received data using the ptr */
-
-                /* alternatively use operator-> */
-                uint32_t counter = sample->counter;
-            })
-            .or_else([](iox::popo::ChunkReceiveResult) { /* handle the error */ });
-    }
-    // wait for new samples (either sleep or use the waitset)
-}
+iox::runtime::PoshRuntime::initRuntime("some_unique_application_name");
 ```
 
-Here we do not check whether we actually have data since we already know there is data available by calling `hasData`.
+The runtime is an object inside the user application that maps the shared memory into the user 
+application's address space. When an application terminates, the runtime cleans up all resources needed for the 
+communication with RouDi. If an application crashes, it also takes care about the clean-up. Due to our 
```

Comment:
oh, you handle the abnormal termination here. But it's RouDi who does the clean-up.